src/main/io/rc_controls.c

```diff
@@ -392,260 +400,126 @@ void applyStepAdjustment(controlRateConfig_t *controlRateConfig, uint8_t adjustm
     }
     switch(adjustmentFunction) {
```

Comment:
this switch can now be replaced by table, with small switch to handle updating of curves ...